src/seth/libexec/seth/seth-calldata

```diff
@@ -12,7 +12,8 @@ if (process.argv.length >= 3 &&
   const sig = process.argv[2];
   try {
     const funcs = new ethers.utils.Interface(["function " + sig.replace(')(', ') returns (')]).functions;
-      console.log(funcs[Object.keys(funcs)[0]].encode(process.argv.slice(3)))
+    const args = process.argv.slice(3).map(s => typeof(s) === 'string' ? JSON.parse(s) : s)
```

Comment:
To make sure I understand: Why are we doing `JSON.parse`? I think [`process.argv`](https://nodejs.org/api/all.html#process_process_argv) returns an array of `string`s. Have you had issues without it?

Reply:
hey @asymmetric , the changes in this PR are meant to address arrays which aren't parsed properly with the trunk version of `seth`.

e.g. `seth calldata 'multicall(bytes[])' '["0x00","0x01"]'`